src/middleware/validation.js

```diff
@@ -0,0 +1,50 @@
+const User = require("../user/model.js");
```

Comment:
If you want to stretch yourself with the validation, then try it with triggers - 

const User = sequelize.define('user', {
  username: {
    type: Sequelize.STRING,
    allowNull: false
  },
  password: {
    type: Sequelize.STRING,
    allowNull: false
  }
}, {
  hooks: {
    beforeCreate: (user, options) => {
      const salt = bcrypt.genSaltSync();
      user.password = bcrypt.hashSync(user.password, salt);
    }
  }
});

It's up to you